mod-source-record-manager/src/main/resources/domain/mod-source-record-manager/features/get-mapping-rules.feature

```diff
@@ -4,8 +4,74 @@ Feature: Source-Record-Storage
     * url baseUrl
     * callonce login testUser
     * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * def marc_bib_rule = read('classpath:samples/marc-bib.json')
```

Comment:
rename marc_bib_rule to marc_bib_rules 
and 
marc_holdings_rule to marc_holdings_rules